documentation/docs/decentralized_identifiers/update.mdx

```diff
@@ -12,19 +12,394 @@ keywords:
+A verification method adds public keys, which can be used to digitally sign things like an identity or a verifiable credential, while a service can provide metadata around the identity via URIs.
 
-## Example
+### Verification Methods
+As demonstrated by the [example](#example) below, the Iota identity framework offers easy to use methods for adding verification methods.
+
+The following properties can be specified for a verification method:
+
+- **id**: a [DID URL](https://www.w3.org/TR/did-core/#did-url-syntax) for the verification method. It can be specified by setting the [fragment](https://www.w3.org/TR/did-core/#fragment);
+- **type**: specifies the type of the Verification Method. The framework supports `Ed25519` and `X25519` key types. This property is automatically filled by the framework when specifying the verification material.
+- **publicKeyMultibase**: multibase encoded public key which concludes the [verification material](https://www.w3.org/TR/did-core/#verification-material). This can be automatically generated by the framework or manually provided by users.
+
+
+### [Verification Relationships](https://www.w3.org/TR/did-core/#verification-relationships)
+Verification Relationships expresses the relationship between the DID subject and the verification method. It can be used to specify the
+ the purpose of the verification method.
+
+The following relationships are supported by the Identity Framework:
+- **[Authentication](https://www.w3.org/TR/did-core/#authentication)**: used to specify authentication methods for the DID subject.
+- **[Assertion](https://www.w3.org/TR/did-core/#assertion)**: can be used for verifiable credential verification.
+- **[Key Agreement](https://www.w3.org/TR/did-core/#assertion)**: used for establishing secure communication channels.
+- **[Capability Invocation](https://www.w3.org/TR/did-core/#capability-invocation)**: can be used to authorize updates to the DID Document.
+- **[Capability Delegation](https://www.w3.org/TR/did-core/#capability-delegation)**: specifies mechanism to delegate cryptographic capability to another party.
+
+Verification methods can be either [embedded in or referenced](https://www.w3.org/TR/did-core/#example-14-embedding-and-referencing-verification-methods). Referencing verification
+methods allows them to be used by more than one verification relationship.
+Upon creating a verification method using the identity framework, specifying the `MethodScope` option will result in an embedded verification method. Leaving that option unset will create the verification method as
+a map entry of the `verificationMethod` property. Verification relationships can be added afterwards using references.
+
+**Note**: Any update to the DID document must be signed using a verification method with `capability invocation` relationship to be valid. Removing all capability invocation verification methods
```

Comment:
Perhaps we should link to a place about the removal of an identity, we do have an example.